documentation/docs/roadmap.md

```diff
@@ -37,7 +37,22 @@ The timeline below shows the past milestones we have completed, along with the f
 * Add support for 0x v3
 
 ## Q1 2020
-* Liquidity Mining Launch: Wave 1 (mid-February)
-* More connectors for decentralized and centralized exchanges: KuCoin, Bitfinex, HitBTC, gate.io
+* Liquidity Mining Launch: Open Beta (February 3, 2020)
+* Liquidity Mining Launch: Wave 1 (mid-March)
+* Released connector for [KuCoin](/connectors/kucoin)
+* More connectors for centralized exchanges: [Bitfinex](https://www.bitfinex.com/), [HitBTC](https://hitbtc.com/), [gate.io](https://www.gate.io/)
+* Improve maintenance of exchange connectors
+* Mock connector testing framework
+* Connector certification program
+
+## Q2 2020
```

Comment:
Add:
* Liquidity mining for [Liquid.com](https://liquid.com) and [KuCoin](https://kucoin.com)

Reply:
Done with requested changes in commit https://github.com/CoinAlpha/hummingbot/pull/1377/commits/796ac078d04a83b3114ac0d72a9990bf4cacd333.